lib/elixir/pages/anti-patterns/macro-anti-patterns.md

```diff
@@ -0,0 +1,15 @@
+# Meta-programming anti-patterns
+
+This document outlines anti-patterns related to meta-programming.
+
+## Unnecessary macros
+
+TODO.
+
+## Large code generation by macros
+
+TODO.
+
+## "use" instead of "import"
+
+TODO (note: include nutrition facts recently added to Library Guidelines)
```

Comment:
```suggestion
TODO

## Large code generation by macros

TODO

## "use" instead of "import"

TODO (+ nutrition facts recently added to Library Guidelines)
```